addons/l10n_sg/models/account_account.py

```diff
@@ -0,0 +1,22 @@
+from odoo import api, models
```

Comment:
Overkill. This box is for statistical purposes and has little to no business impact.